newdle/client/src/selectors.js

```diff
@@ -15,6 +15,7 @@ export const isAcquiringToken = state => !!state.auth.acquiringToken;
 
 // user
 export const getUserInfo = state => state.user;
+export const getUserTimezone = _state => moment.tz.guess();
```

Comment:
I'll leave it as `export const getUserTimezone = () => moment.tz.guess();`. In the original issue it was discussed that it would be nice to allow the user to change this timezone, and I agree. So I want to leave it like this so a future maintainer doesn't need to worry about deprecating and removing function on `util/date.js`

Reply:
good point! i wasn't thinking about that case but it makes sense to store the custom timezone in the state so keeping the selector is indeed a good idea :)